protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchemaUtils.java

```diff
@@ -202,4 +201,54 @@ private static void appendIndented(StringBuilder sb, String value) {
               .append('\n');
     }
   }
+
+  private static String toOptionString(OptionElement obj) {
```

Comment:
Let's make the signature `private static String toString(OptionElement type)` to be consistent with the other methods